### PR TITLE
Use the new gdp-hmi

### PR DESCRIPTION
**Pull request by RobertAJMarshall**
[GDP-279] Update copyright on HMI applications

### CODE REVIEW

File: meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/genivi-dev-platform-hmi.inc [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Copyright (C) 2015 GENIVI Alliance
+# Copyright (C) 2016 GENIVI Alliance
```
Comment (chbae) [inline meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/genivi-dev-platform-hmi.inc]: - # Copyright (C) 2015-2016 GENIVI Alliance

**Comment by chbae** [comment]
This is really new HMI? If yes, or I'm confusing title in commit message.

File: meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/genivi-dev-platform-hmi.inc (hunk 1 above)
Comment (RobertAJMarshall) [inline meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/genivi-dev-platform-hmi.inc]: OK

**Comment by RobertAJMarshall** [comment]
Would you prefer the (text) latest gdp-hmi?

**Comment by chbae** [comment]
I prefer to using below style title.
ex) gdp-hmi: Fix dates

**Comment by tom--pollard** [comment]
Could you update the title of this commit @RobertAJMarshall ?

**Pull request closed without merging**